From 5a24a604ec2dbb26d91cb4c376f56097c6f83792 Mon Sep 17 00:00:00 2001
From: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
Date: Tue, 16 Jul 2019 15:25:59 +0700
Subject: [PATCH 151/328] arm64: dts: renesas: r8a774c0-cat874: Define and
 enable Bluetooth pin.

This patch defines and enables pin for bluetooth device
used for RZ/GE board (R8A774C0 SoC).

Signed-off-by: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
index 3bde247..7a6c9dc 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-cat874.dts
@@ -492,6 +492,15 @@
 	status = "okay";
 };
 
+&gpio4 {
+	bluetooth-en-gpio {
+		gpio-hog;
+		gpios = <6 GPIO_ACTIVE_HIGH>;
+		output-high;
+		line-name = "bluetooth-en-gpio";
+	};
+};
+
 &du {
 	pinctrl-0 = <&du_pins>;
 	pinctrl-names = "default";
-- 
1.9.1

